capi/Makefile.am

```diff
@@ -17,6 +17,7 @@ libyami_capi_ldflags = \
 
 libyami_capi_cppflags = \
 	$(LIBVA_CFLAGS) \
+	-I$(top_srcdir)/interface \
```

Comment:
please check you're doing the same that the rest of the libraries when including the interface headers? Otherwise you'll break standard inclusion way